.github/workflows/tests-e2e-upgrade.yaml

```diff
@@ -585,6 +588,24 @@ jobs:
             --include-conn-disrupt-test \
             --conn-disrupt-test-restarts-path "./cilium-conn-disrupt-restarts" \
             --flush-ct \
+            --test "seq-.*" \
```

Comment:
I haven't been recently following the CLI development. What is this `seq-.*`? Grep does not find any relevant occurrences.

Reply:
`--test "seq-.*" ` - won't this prevent from testing `no-interrupted-connections`?

Reply:
@brb Thanks for the review. After the last changes I'm seeing
```
[=] [cilium-test-1] Test [no-interrupted-connections] [1/105]
```

for the lines that are exclusively running the `--test "no-interrupted-connections"`.

I assume this is what we want?